src/makeRobot.js

```diff
@@ -37,8 +37,53 @@
  *
  * @return {object}
  */
-function makeRobot(name, wheels, version) {
-  // write code here
+function makerobot(name, wheels, version) {
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    evacuate() {
```

Comment:
```suggestion
    },
    
    evacuate() {
```
add empty line between methods for readability purposes